File: admin/src/main/java/com/beat/admin/user/facade/AdminUserFacade.java
package com.beat.admin.user.facade;

import org.springframework.stereotype.Service;

import com.beat.admin.user.api.response.UserFindAllResponse;
import com.beat.admin.user.application.query.AdminUserQueryService;

import lombok.RequiredArgsConstructor;

@Service
@RequiredArgsConstructor
public class AdminUserFacade {

	private final AdminUserQueryService adminUserQueryService;

	public UserFindAllResponse checkMemberAndFindAllUsers(Long memberId) {
		return UserFindAllResponse.from(adminUserQueryService.findAllUsers(memberId));
	}
}

File: admin/src/main/kotlin/com/beat/admin/user/facade/AdminUserFacade.kt
package com.beat.admin.user.facade

import com.beat.admin.user.api.response.UserFindAllResponse
import com.beat.application.admin.user.query.AdminUserQueryService
import org.springframework.stereotype.Service

@Service
class AdminUserFacade(
    private val adminUserQueryService: AdminUserQueryService,
) {
    fun checkMemberAndFindAllUsers(memberId: Long): UserFindAllResponse =
        UserFindAllResponse(adminUserQueryService.findAllUsers(memberId))
}
